src/dashboard.py

```diff
@@ -467,6 +467,9 @@ def __init__(self, name: str = None, data: dict = None, backup: dict = None, sta
         self.Name = name
         self.__configPath = os.path.join(DashboardConfig.GetConfig("Server", "wg_conf_path")[1], f'{self.Name}.conf')
         
+        if not os.path.exists(os.path.join(DashboardConfig.GetConfig("Server", "wg_conf_path")[1], 'WGDashboard_Backup')):
```

Comment:
Perhaps declare the variable instead of repeating the same LONG line twice? What is your opinion? If you disagree feel free to resolve.